js/a11y.js

```diff
@@ -438,13 +438,16 @@ class A11y extends Backbone.Controller {
     const config = this.config;
     $element = $($element).first();
 
+    const isInDOM = Boolean($element.parents('body').length);
+    if (!isInDOM) return false;
+
+    const isOutsideOpenPopup = this.isPopupOpen && !this.popupStack?.lastItem[0]?.contains($element[0]);
+    if (isOutsideOpenPopup) return null;
```

Comment:
Minor: Probably best to return Boolean false here

Reply:
This function intentionally returns `true`, `false` or `null` for where it should return a definitive response at the given element (`true` or `false`) or allow descending into the element's children (`null`).

The function `_findFirstForwardDescendant` can take `isReadable` as its selector. These lines are the important ones:

https://github.com/adaptlearning/adapt-contrib-core/blob/a1b6c57f8608d0a38ef6918ce3bc9bad6961f580/js/a11y.js#L666-L675

Where an element is outside of the open popup, the element may be an ancestor of the open popup and therefore its descendants should be searchable such that we can find the first `selector` element in the document.

Returning `null` is intended.